.github/workflows/canary-integration-test.yml

```diff
@@ -1708,64 +1713,6 @@ jobs:
           name: ${{ github.job }}-${{ matrix.ceph-image }}
           additional-namespace: kube-system
 
-  csi-hostnetwork-disabled:
```

Comment:
I'm pretty sure that this test only existed because holder pods used to be enabled when hostnetwork was disabled. I think we also decided not to support this case officially, only as a reasonable-effort stance. Without holders, and without documented support, I don't see a reason to keep this test.